app/presenters/formats/service_sign_in_presenter.rb

```diff
@@ -0,0 +1,129 @@
+module Formats
+  class ServiceSignInPresenter
```

Comment:
How are you planning to invoke this when you have a new yaml file?

Reply:
We're planning to create a rake task to handle publishing this format, and we'll probably pass the file path as an argument to the rake task, load the file and then send it as an argument to ServiceSignInPresenter.

Reply:
👍